ada/language/ast.py

```diff
@@ -8904,7 +8904,19 @@ class TargetName(Name):
     Name for Ada 2020 ``@``.
     """
 
-    pass
+    # NOTE: This is useless for us for TargetName, but the Ada RM decided to
+    # classify '@' as a name, even though it is non obvious to link this to a
+    # declaration.
```

Comment:
I guess you considered the option of linking it to the AssignStmt.dest's declaration, what's your rationale for not doing so, by curiosity?